package.json

```
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "npm-run-all --parallel server client",
+    "start": "npm-run-all --parallel client",
```

If you just work with a client, you could just run `npm run client` instead of `npm start` OR clean this up. The way this is written, it makes no sense (running *just one thing* in parallel ...).
```suggestion
    "start": "npm run client",
```